scripts/prepare_network.py

```diff
@@ -151,7 +140,7 @@ def average_every_nhours(n, offset):
     return m
 
 
-def apply_time_segmentation(n, segments):
+def apply_time_segmentation(n, segments, solver_name="cplex"):
```

Comment:
Shouldn't we use an open-source solver as default, like cbc?

Reply:
yup, already changed in master apparently.